Doc/library/sys.rst

```diff
@@ -378,26 +378,41 @@ always available.
    .. versionadded:: 3.8
       __unraisablehook__
 
+
+.. function:: exception()
+
+   This function returns the exception that is currently being handled.  This
```

Comment:
It's not directly obvious what you mean with ``the exception``. I assume it's the exception object and equivalent to ``sys.exc_info()[1]`` ?

Reply:
Yes, that's what I mean. 

I changed the doc so that exc_info() is described in terms of exception(). In the future there should be less need for people to know what exc_info() does, so I don't think this should be defined in terms of exc_info().